common/appscale/common/monit_interface.py

```diff
@@ -408,3 +410,15 @@ def _reload(self):
     yield gen.sleep(wait_time)
     self.last_reload = time.time()
     subprocess.check_call(['monit', 'reload'])
+
+  @staticmethod
+  def _parse_security_token(http_response):
+    """ Extracts the security token from an HTTP response.
+
+    Args:
+      A tornado.httpclient.HTTPResponse object.
+    Returns:
+      A string containing the security token.
+    """
+    cookie = SimpleCookie(http_response['Set-Cookie'])
+    return cookie['securitytoken'].value
```

Comment:
KeyError can be thrown here, if there are no cookies in the response, and if securitytoken isn't a cookie. I think the semantics should be that we return None if the token isn't found:
```
try:
    cookie = SimpleCookie(http_response['Set-Cookie'])
    return cookie['securitytoken'].value
except KeyError:
    return None
```